AuthenticationApp/Controller/SignUpVC.swift

```diff
@@ -129,15 +136,41 @@ private extension SignUpVC {
     }
     
     func saveUserData() {
+        saveImage()
         UserDefaults.standard.set(nameTextField.text, forKey: "name")
         UserDefaults.standard.set(emailTextField.text, forKey: "email")
         UserDefaults.standard.set(enterPasswordTextField.text, forKey: "password")
         UserDefaults.standard.set(genderLabel.text, forKey: "gender")
     }
     
+    func saveImage() {
+        guard let data = profileImageView.image?.jpegData(compressionQuality: 0.5) else { return }
+        UserDefaults.standard.set(data, forKey: "profileImage")
+    }
+    
     func goToLogin() {
         if let loginVC = storyboard?.instantiateViewController(identifier: "LoginVC") {
             navigationController?.setViewControllers([loginVC], animated: true)
         }
     }
 }
+
+extension SignUpVC: UINavigationControllerDelegate, UIImagePickerControllerDelegate {
+    
+    func presentImagePicker() {
```

Comment:
This function is not part of the delegate protocol, it should not be part of this extension.

Reply:
Done

Reply:
Done